### DISCUSSION TITLE
Blish Builds

### DISCUSSION THREAD

**Discussion by KenediaDev** [blish-hud/Blish-HUD#648] [Module Ideas]
### Build and Gear templates
Not a real suggestion as I plan to do it myself, but would love to collect some input.

- create builds and gear sets like you do on gw2skills
- Build part of the template will offer a way to copy the build template code to quickly load them into your actual build
- Gear part of the template will provide two benefits

1. Save gear setups to look what stats etc beeing used with that build
2. All different stats and upgrades will be listed with a button to copy their names to quickly setup their gear through the **Legendary Armory**

**Comment by Snaid1** [comment]
I'm in the early stages of doing a build storage tab for Blishpad as a separate module (going to be awhile before I pre-release.) I've got no plans on having a build creator as part of it but it might be cool to collaborate depending on what you are planning. I don't have plans for anything regarding Equipment templates since you can't copy those into notepad but maybe we could do something like if they have my build storage and your build designer a button is visible that opens the designer from storage?

**Reply by KenediaDev** [reply]
You mean just to load your templates into my build manager?

At what point are you? I want to create a complete solution
- create
- store
- "load" 
Builds and Gear

**Reply by Snaid1** [reply]
Still really early. All I've done basically is start converting the Javascript Build Template parsing code from ThatShaman into C#. if you want to do an all in one solution go for it. I can back off and not do template storage. 

here is a link to ThatShaman's JS file if you want to reference it: https://github.com/thatshaman/Buildtemplate/blob/master/Buildtemplate.js

**Reply by KenediaDev** [reply]
Thanks a lot for the link!
Well that's quite a big project and I will move next weekend. I'm not that sure how fast I would be done.
I don't mind if there are two modules etc.

No need to back off if you would like to create it anyway from my side.

**Reply by Snaid1** [reply]
If you are making an all in one maybe I'll just add basic build storage to blishpad directly instead of it's own module and skip some of the fancy features I thought about implementing. That way if all people want is basic build storage Blishpad will work but if they want something more then they'll need to download your module.

**Comment by Snaid1** [comment]
If you are making an all in one maybe I'll just add basic build storage to
blishpad directly and skip doing some fancy features I thought of doing.
That way if all people want is build template storage they have blishpad
but if they want anything more they would need your module.

**Reply by KenediaDev** [reply]
This is just an accidental comment I guess?

**Reply by Snaid1** [reply]
or something, I was asleep and logged out of github when it posted, and it's copy and paste from my last comment on our earlier thread on this discussion. So it may have been some sort of automation on githubs part but I think I'm going to reset my password to be on the safe side.
